zipline/pipeline/expression.py

```diff
@@ -328,7 +330,12 @@ def __repr__(self):
 
     def short_repr(self):
         """Short repr to use when rendering Pipeline graphs."""
-        return "Expression: {expr}".format(
+        final = self._expr
+        num_list = re.findall(r"[-+]?\d*\.\d+", self._expr)
```

Comment:
Is the purpose of this to replace numbers with scientific notation? If so, do we need to use `Decimal` here (vs. say, `float`)?

Reply:
No, that was just from the original example of shortening numbers I found - I've changed it to `float`